src/h3/mod.rs

```diff
@@ -2288,21 +2289,23 @@ mod tests {
     }
 
     #[test]
-    /// Client sends data on a QPACK stream, which we don't allow.
+    /// Client closes QPACK stream, which is forbidden
     fn close_qpack_stream() {
-        // TODO: once we add support for QPACK dynamic table, this test should
-        // be changed to testing for critical stream being closed.
         let mut s = Session::default().unwrap();
         s.handshake().unwrap();
 
         let mut qpack_stream_closed = false;
 
-        s.send_frame_client(
-            frame::Frame::MaxPushId { push_id: 1 },
-            s.client.local_qpack_streams.encoder_stream_id.unwrap(),
-            true,
-        )
-        .unwrap();
+        s.pipe
+            .client
+            .stream_send(
+                s.client.local_qpack_streams.encoder_stream_id.unwrap(),
```

Comment:
Might be a good idea to save this in a separate variable, so we would avoid the odd formatting, e.g.
```
let stream_id = s.client.local_qpack_streams.encoder_stream_id.unwrap();
s.pipe.client.stream_send(stream_id, ...).unwrap();
```